src/main/java/com/volasoftware/tinder/jwt/JwtGenerator.java

```diff
@@ -0,0 +1,24 @@
+package com.volasoftware.tinder.jwt;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+
+import java.util.Date;
+
+public class JwtGenerator {
+    private static final String SECRET_KEY = "UnbreakableSecretKey";
+    private static final long EXPIRATION_TIME =604800000L;
```

Comment:
```suggestion
    private static final long EXPIRATION_TIME = 604800000L;
```
How long is that?
Is it normal to be same value to each environment?